libkrb5/src/context.rs

```diff
@@ -527,23 +541,27 @@ impl Krb5Context {
         Ok(plain)
     }
 
-    fn parse_wrap_token(encoded_data: &[u8], usage: Krb5KeyUsage, seq_num: i32) -> Result<(Vec<u8>, Vec<u8>), Krb5Error> {
+    fn parse_wrap_token(encoded_data: &[u8], usage: Krb5KeyUsage, seq_num: Option<i32>) -> Result<(Vec<u8>, Vec<u8>), Krb5Error> {
         let (mut header, cipher_text) = (encoded_data[..16].to_vec(), &encoded_data[16..]);
         let rrc = Krb5Context::parse_and_verify_wrap_token_header(header.as_slice(), usage, seq_num).unwrap();
         let cipher_text = Krb5Context::rotate_left(cipher_text, rrc);
 
         Ok((cipher_text, header))
     }
 
-    fn parse_and_verify_wrap_token_header(header: &[u8], usage: Krb5KeyUsage, expected_seq_num: i32) -> Result<u16, Krb5Error> {
+    fn parse_and_verify_wrap_token_header(header: &[u8], usage: Krb5KeyUsage, expected_seq_num: Option<i32>) -> Result<u16, Krb5Error> {
```

Comment:
It seems this is the same function as verify_mic_token_header() if I am not mistaken.